other/upgrade_2-1_mysql.sql

```diff
@@ -1908,4 +1880,28 @@ DROP INDEX idx_topic on {$db_prefix}messages;
  
 ---# duplicate to topics_last_message_sticky and topics_board_news
 DROP INDEX idx_id_board on {$db_prefix}topics;
----#
\ No newline at end of file
+---#
+
+/******************************************************************************/
+--- update ban ip with ipv6 support
+/******************************************************************************/
+
+ALTER TABLE {$db_prefix}ban_items ADD COLUMN ip_low varbinary(16);
+ALTER TABLE {$db_prefix}ban_items ADD COLUMN ip_high varbinary(16);
+
+UPDATE {$db_prefix}ban_items
+SET ip_low = 
+    UNHEX(
+        hex(
```

Comment:
I don't know another way to convert this in varbinary.
the hex convert the int to hex,
the unhex convert this to varbinary.